js/okex.js

```diff
@@ -1835,6 +1835,82 @@ module.exports = class okex extends Exchange {
         return this.parseOrder (order, market);
     }
 
+    async cancelOrders (ids, symbol = undefined, params = {}) { // TODO : the original endpoint signature differs. You can skip individual symbol and assign them in batch
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' canelOrders() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = [];
+        // for ids
+        if (typeof ids === 'string') {
+            const orderIds = ids.split (',');
+            for (let i = 0; i < orderIds.length; i++) {
+                request.push ({
+                    'instId': market['id'],
+                    'ordId': orderIds[i],
+                });
+            }
+        } else {
+            for (let i = 0; i < ids.length; i++) {
+                request.push ({
+                    'instId': market['id'],
+                    'ordId': ids[i],
+                });
+            }
+        }
+        // for order-ids
+        let orderId = this.safeValue2 (params, 'ordIds', 'orderIds');
```

Comment:
I don't think that we should process ids and orderIds. It's either `ids` or [`ordIds`+`orderIds`]. This exchange does not have any mention of `orderIds` or `ordIds` in their docs, so these two parameters don't belong here.

Another aspect here – if clientOrderIds are specified – we don't sent `ids` instead, otherwise we send clientOrderIds.

Reply:
I've removed 'ordId' at all, and now method only checks existence of `clOrdId`, and if they are not provided, then checks 'ids' argument